gateway/v10.ts

```diff
@@ -2098,7 +2098,15 @@ export type GatewayVoiceStateUpdateDispatch = _DataPayload<
 /**
  * @see {@link https://discord.com/developers/docs/topics/gateway-events#voice-state-update}
  */
-export type GatewayVoiceStateUpdateDispatchData = APIVoiceState;
+export type GatewayVoiceStateUpdateDispatchData = Omit<APIVoiceState, 'member'> & {
```

Comment:
Tmk we wanna avoid Omits and use interfaces instead, so maybe `export interface x  extends`